app/assets/javascripts/blacklight/blacklight.js

```diff
@@ -441,7 +441,7 @@ Blacklight.doSearchContextBehavior = function () {
   const nodes = Array.prototype.slice.call(elements);
   nodes.forEach(function (element) {
     element.addEventListener('click', function (e) {
-      Blacklight.handleSearchContextMethod.call(e.target, e);
+      Blacklight.handleSearchContextMethod.call(e.currentTarget, e);
```

Comment:
Can we do this here: https://github.com/projectblacklight/blacklight/blob/edca682fcf34c949ade403bc756d5df89b6d7cd4/app/javascript/blacklight/search_context.js#L13 and then use the build thing?

```sh
$ npm run js-compile-bundle
```

Reply:
Ah, thanks. I forgot that's how we're doing things now.